editor/gui/editor_scene_tabs.cpp

```diff
@@ -341,6 +341,8 @@ void EditorSceneTabs::_bind_methods() {
 EditorSceneTabs::EditorSceneTabs() {
 	singleton = this;
 
+	set_process_shortcut_input(true);
```

Comment:
Unsure where to place this, i.e. if the order matters, can move if it does matter